pkg/query-service/app/http_handler.go

```diff
@@ -2188,6 +2189,124 @@ func (aH *APIHandler) WriteJSON(w http.ResponseWriter, r *http.Request, response
 	w.Write(resp)
 }
 
+// Preferences
+
+func (ah *APIHandler) RegisterPreferenceRoutes(router *mux.Router, am *AuthMiddleware) {
```

Comment:
the routes do not follow the REST semantics.

Reply:
as not all the functionalities are exposed to the UI and UI just controls the getters and setters. The insert is taken care by the bootstrap entity / updates and deletes is taken care by the migration. Since it's for all tenants we won't be using HTTP calls for the same .